src/main/java/ee/tuleva/onboarding/comparisons/returns/ReturnRateAndAmount.java

```diff
@@ -0,0 +1,6 @@
+package ee.tuleva.onboarding.comparisons.returns;
+
+import ee.tuleva.onboarding.currency.Currency;
+import java.math.BigDecimal;
+
+public record ReturnRateAndAmount(BigDecimal rate, BigDecimal amount, Currency currency) {}
```

Comment:
I usually prefer to avoid using the word "and" in any method names and class names. It's usually a code smell (a violation of the Single Responsibility Principle). I.e. a class that does more than one thing (and could be split into 2) or a method that does more than 1 thing (and could be split).

Reply:
good point

Reply:
Fixed